concent_api/conductor/tests/test_integration_conductor.py

```diff
@@ -210,6 +210,53 @@ def test_conductor_should_not_schedule_verification_order_task_if_it_was_already
             self.assertEqual(UploadReport.objects.count(), 3)
             self.assertEqual(mock_task.call_count, 1)
 
+    def test_conductor_should_schedule_verification_order_task_if_it_uploaded_file_path_was_not_existing_before(self):
```

Comment:
Basically, I would say we need 2 tests:
1. same file is uploaded twice -> no `upload_finished` is scheduled
2. same file is uploaded twice (i.e. source) and then the second one is uploaded (i.e. result) -> `upload_finished` is scheduled

Reply:
:+1: done